pom.xml

```diff
@@ -71,27 +71,27 @@
   </ciManagement>
 
   <properties>
-    <jetty.version>9.4.49.v20220914</jetty.version>
-    <jersey.version>2.37</jersey.version>
-    <jackson.version>2.13.4</jackson.version>
-    <jackson-databind.version>2.13.4.2</jackson-databind.version>
+    <jetty.version>9.4.53.v20231009</jetty.version>
+    <jersey.version>2.41</jersey.version>
+    <jackson.version>2.14.3</jackson.version>
```

Comment:
any reason we cant go to 2.15.xx of jackson?

Reply:
2.15.x has limits on the max json size that requires further code changes in the servers.